### PR TITLE
gl_common: unify gd definition

### PR DESCRIPTION
**Pull request by mighty9245**
let's have this named and defined everywhere equally?

### CODE REVIEW

File: src/backend/gl/gl_common.c [hunk 1]
```diff
@@ -982,7 +982,7 @@ void gl_destroy_blur_context(backend_t *base attr_unused, void *ctx) {
  */
 void *gl_create_blur_context(backend_t *base, enum blur_method method, void *args) {
 	bool success = true;
-	auto gd = (struct gl_data *)base;
+	struct gl_data *gd = (void *)base;
```
Comment (yshui) [inline src/backend/gl/gl_common.c]: I prefer to use `auto` here.

File: src/backend/gl/gl_common.c [hunk 2]
```diff
@@ -1376,7 +1376,7 @@ static void gl_image_apply_alpha(backend_t *base, struct gl_image *img,
 }
 
 void gl_present(backend_t *base, const region_t *region) {
-	auto gd = (struct gl_data *)base;
+	struct gl_data *gd = (void *)base;
```
Comment (yshui) [inline src/backend/gl/gl_common.c]: ditto

**Comment by mighty9245** [comment]
got you

**Comment by mighty9245** [comment]
can we have gd defined everywhere in this way? `auto gd = (struct gl_data *)base;`

**Comment by yshui** [comment]
@mighty9245 Yes. That will be nice.

**Comment by mighty9245** [comment]
ok, i'll make new pull request soon since i messed with this one a little :)

**Pull request closed without merging**